app/cypress/e2e/cif/project-revision/create-first-project-revision.cy.js

```diff
@@ -466,6 +488,7 @@ describe("when creating a project, the project page", () => {
     cy.findByText(/Review and Submit information/i).click();
     cy.contains("Review and Submit Project").should("be.visible");
     cy.findByRole("button", { name: /^discard/i }).click();
+    cy.wait(1000);
```

Comment:
Can we wait on something better, like screen on text? We basically need to wait for the previous click to complete the re-render